scipy/optimize/tests/test_linprog.py

```diff
@@ -1648,6 +1648,7 @@ def test_bug_10466(self):
         """
         Test that autoscale fixes poorly-scaled problem
         """
+        pytest.skip()
```

Comment:
The standard way of doing this is as a decorator, and using `reason=`.

Reply:
Sorry, I was not clear here. I am not sure this should be done at all. I committed this to be sure it was the reason CI was not going. There is already a skipif so if we do want to skip this entirely, then I would rewrote this one.